functions/utils/application.js

```diff
@@ -0,0 +1,69 @@
+const createAppObject = (body) => {
+  const appObj = {
+    title: body["title"],
+    email: body["email"],
+  };
+  return appObj;
+};
+
+const validateAppData = (data) => {
+  const fields = [];
+  let errors = [];
+
+  fields.forEach((key) => {
```

Comment:
we initialize fields at line 10 as an empty array and then call `fields.forEach((key) => {`. Did you mean to assign `fields = data` at line 10?

Reply:
So fields should be updated with the names of the fields of appObj from the above function. Since we don't have the app schema and the client side request format created I have appObj as IP. For example for us since appObj will have the field names of title and email we would populate that array with those names. I'll definitely update this with comments for clarity to help the developer who will be finishing this story!

Reply:
ahhhh sorry! sounds good